CHANGES.rst

```diff
@@ -29,6 +29,7 @@ Bugfix:
  - Fix notification issue when Riot is not started (#2451)
  - Fix Unable to add Matrix apps (#2466)
  - Riot auto joined a public room (#2472)
+ - Disable Firebase analytics (#2481)
```

Comment:
Could we replace this by "Remove last traces of Firebase analytics" ?

We stopped using and sending analytics to GA a while ago. This PR is more code cleaning.